### ISSUE TITLE
No module named 'gi'

### ISSUE THREAD

**Issue by magdalenat** [ContinuumIO/anaconda-issues#849]
My system is Win8 Pro 64 bit.

I need in my script a line: `from gi.repository import Gtk`
Attempts of running script are as follows:

    C:\>python
    Python 3.4.4 |Continuum Analytics, Inc.| (default, Feb 16 2016, 09:54:04) [MSC v.1600 64 bit [\cut]
    Type "help", "copyright", "credits" or "license" for more information.
    >>> exit()

    C:\>python "d:\!_BitBucket\PyCodeMisc\polarPlot.py"
    Traceback (most recent call last):
    File "d:\!_BitBucket\PyCodeMisc\polarPlot.py", line 3, in <module>
    from gi.repository import Gtk
    ImportError: No module named 'gi'

    C:\>Python34\python.exe "d:\!_BitBucket\PyCodeMisc\polarPlot.py"
    d:\!_BitBucket\PyCodeMisc\polarPlot.py:3: PyGIWarning: Gtk was imported without specifying [\cut]    
    Use gi.require_version('Gtk', '3.0') before import to ensure that the right version get
    from gi.repository import Gtk
    C:\>

In short: I can run my script when I use separate Python installation, ater installing PyGI/PyGObject from [here](https://sourceforge.net/projects/pygobjectwin32/files/)

I do not know how to install this into Anaconda. I have tried pip, conda with diggerent name of modules I found in different answers concerning "ImportError: No module named 'gi'", but it still does not work with Anaconda.
How can I set it up ?

Regards

**Comment by adamziek** [comment]
Also having an identical problem. I'm trying to run code which uses PyGI/ PyGObject but have no idea which packages need to be installed via conda in order for it to load the gi module.   I've had the code running on a Windows machine with PyGObject installed - no problems there.

Any help would be most appreciated!

Adam

**Comment by ccordoba12** [comment]
As far as I know, there are no conda packages for this module, sorry.

**Comment by adamziek** [comment]
Thanks. I've been trying to find a way to compile the PyGI / GTK+ libraries from source to work in a virtual python environment (e.g. PythonBrew or Anaconda) but with little success yet.. If anyone has any ideas, I'd very much appreciate it!

**Comment by mingwandroid** [comment]
For `conda` packages, we (and @magdalenat) would want them to work on Windows if possible and also on Python 2.7, 3.4 and 3.5. It may be worth seeing if `conda-forge` would be interested in PyGI/PyGTK packages for Linux and OS X, then later tackling Windows which would be a large undertaking (and compiling these packages for Windows Python 2.7 may not be possible due to MSVC 2008's lack of C99 support). A good starting point would be https://github.com/wingtk/gtk-win32

**Comment by adamziek** [comment]
Out of interest, I've installed (via conda) the PyGObject2 and GTK2 packages from user mw - but still cannot access the gi module, i.e.

> import gi   

ImportError: No module named gi

I realise you don't support these packages 'out-of-the-box', but should there be a reason these 3rd party additions aren't recognised? Conda installs them without error..

(I'm a bit of a newbie at this)

Thanks again

**Comment by mingwandroid** [comment]
Yes, Binary Compatibility must be considered carefully.

**Comment by PyPatel** [comment]
I am currently having the same issue. Did anyone have solution to this.? Please post it.
